parking-spot-submodule/src/main/java/ua/lviv/iot/application/repository/ParkingSpotRepositoryOutPort.kt

```diff
@@ -15,7 +14,7 @@ interface ParkingSpotRepositoryOutPort  {
 
     fun update(entity: ParkingSpot): Mono<ParkingSpot>
 
-    fun deleteById(id : String): Mono<DeleteResult>
+    fun deleteById(id : String): Mono<Void>
```

Comment:
it's better to have Mono<Unit> instead of Mono<Void>

Reply:
What do you mean?🤓